zhaquirks/senoro/senorowin.py

```diff
@@ -0,0 +1,43 @@
+"""Sensoro Window Sensor (TS0601)."""
+
+from zigpy.quirks.v2.homeassistant import EntityPlatform, EntityType
+import zigpy.types as t
+
+from zhaquirks.tuya import BatterySize
+from zhaquirks.tuya.builder import TuyaQuirkBuilder
+
+
+class OpeningStateEnum(t.enum8):
+    """Enum for opening state."""
+
+    open = 0
+    closed = 1
+    tilted = 2
```

Comment:
Let's make these uppercase to follow other enums:

```suggestion
    Open = 0
    Closed = 1
    Tilted = 2
```

Due to some ZHA limitations, these are not translateable for now, but we'll work on changing that in the future.
Related PR: https://github.com/zigpy/zha/pull/86

Reply:
Done